### ISSUE TITLE
A1: vocalisation acoustics pilot

### ISSUE THREAD

**Issue by in-c0** [in-c0/cat-pose-benchmark#19]
## Research question

Which reproducible acoustic features of feline vocalisations predict observable context, subsequent behaviour, or response to human action better than simple statistical/context baselines?

This thread does **not** assume that a meow maps to an English phrase. The first goal is to establish whether audio contains information that survives identity/session controls and calibrated held-out evaluation.

## Evidence base informing the pilot

A useful existing research-only starting point is **CatMeows v1.0.2 (Zenodo record 4008297)**: 440 meows from 21 cats across three elicitation contexts — brushing, waiting for food, and isolation in an unfamiliar environment. Its stated terms are scientific-research/non-commercial, so it must remain licence-separated from any later commercial product training.

The associated 2019 study by Ntalampiras et al., *Automatic Classification of Cat Vocalizations Emitted in Different Contexts* (Animals 9(8):543, DOI 10.3390/ani9080543), used MFCC and temporal-modulation features to classify emission context. That gives A1 a concrete replication baseline rather than a blank-slate deep model.

A 2025 Animal Behaviour study, *Acoustic classification and human perception of domestic cat vocalizations* (224:123204), clustered 212 vocalisations using 10 acoustic parameters and found four acoustically distinct but substantially graded classes. That is a useful warning not to equate folk labels such as `meow`, `chirp`, or `yowl` with clean biological categories.

Classical feline-acoustics work also supports explicit measurement of F0, spectral-energy structure, duration, and formant-like resonances; articulation can materially alter these features. These measurements should therefore be retained even if learned embeddings are added later.

## A1.0 — zero-new-collection replication

Start without collecting any new animal data.

### Dataset

Use CatMeows only as a research baseline, preserving its original licence and attribution requirements. Do not redistribute its audio through this repository unless the dataset terms are separately reviewed and permit that exact release.

### Primary target

Replicate **emission-context prediction** for the three supplied contexts. This is an observable experimental condition, not an intent label.

### Leakage-safe splits

Random clip-level splitting is prohibited.

- group repeated calls from the same recording sequence/session;
- no cat may appear in both train and test in the primary cross-cat result;
- report within-cat results separately if studied;
- inspect owner/breed confounding explicitly, because several cats share owner/breed structure;
- fit scaling, feature selection, calibration, and hyperparameters on training folds only;
- preserve one frozen final test split or nested grouped CV protocol before model comparison.

### Baselines

At minimum:

1. majority/prevalence prior;
2. context prior conditioned on available metadata, reported as a confound baseline;
3. cat-identity / breed / sex metadata baselines where available;
4. duration + F0 summary baseline;
5. MFCC baseline;
6. MFCC + temporal-modulation replication baseline;
7. one compact learned-audio embedding baseline only after the interpretable baselines are frozen.

A model that cannot beat metadata/context/confound baselines does not advance merely because its raw accuracy looks high.

### Acoustic feature families

Preserve both interpretable and representation-learning tracks.

**Call-level / interpretable**
- duration;
- F0 median/range/slope/contour statistics;
- voiced/unvoiced fraction;
- spectral centroid, bandwidth, rolloff and peak frequency;
- harmonicity/HNR or equivalent periodicity measures;
- formant-like/resonance measurements where estimator confidence is adequate;
- roughness/noisiness measures;
- RMS/amplitude-envelope descriptors, treated cautiously because gain and microphone placement can confound them;
- repetition/inter-call timing when sequence information is available.

**Frame/representation**
- MFCCs + deltas;
- temporal-modulation features;
- log-mel representation;
- later: one frozen pretrained audio embedding for comparison, not as the only model.

### Metrics

Report more than accuracy:

- balanced accuracy;
- macro F1;
- per-class precision/recall;
- log loss;
- Brier score where probabilities are comparable;
- calibration curve / ECE;
- confusion matrix;
- bootstrap or grouped-resampling confidence intervals where feasible.

## A1.1 — acoustic structure before semantics

Run an unsupervised analysis independent of the supplied context labels.

Questions:

- Do stable acoustic clusters recur under resampling?
- Are clusters dominated by individual identity rather than context?
- How much mutual/predictive information do clusters carry about context after identity controls?
- Do human folk call labels, if later added, map cleanly to acoustic clusters or cut across them?

This stage may emit latent acoustic states such as `A-Z17`; it must not immediately rename them as human-language intents.

## A1.2 — prospective naturalistic household pilot

Only after A1.0/A1.1 and the shared event contract in #18 are stable.

Do **not** recreate stressful elicitation paradigms merely to obtain labels. The prospective pilot should begin with naturally occurring, ordinary household episodes such as routine feeding preparation, voluntary play, greeting/attention, or requests around already-permitted access.

Required capture:

- raw audio plus exact recording settings;
- event/session/cat pseudonym;
- observable location/object state;
- naturally occurring human actions;
- what happens next;
- signalling continuation/termination window;
- explicit missing modalities;
- privacy/consent metadata for any human audio.

Use the #18 intent-event record when available.

## Outcome targets for prospective data

Prefer targets that can be observed without mind-reading:

- next coarse behaviour/action;
- approach/withdrawal;
- object/location transition;
- signalling continuation vs termination;
- response to a routine, low-risk human action;
- event duration / recurrence.

Human intent labels can be analyzed secondarily but are not primary ground truth.

## Acceptance gates

1. Acoustic models are compared against strong metadata/context/statistical baselines.
2. Primary evaluation predicts observable context/outcome, not annotator storytelling.
3. Same-cat/session/sequence leakage is controlled and documented.
4. Cross-cat and within-cat performance are reported separately.
5. Calibration is reported alongside discrimination metrics.
6. Learned embeddings are ablated against interpretable acoustic features.
7. Identity/breed/owner confounding is quantified where metadata permits.
8. `unknown/other` remains a legal state in prospective work.
9. Failure to beat baselines is publishable evidence.
10. No dataset with research-only/non-commercial terms is silently reused for commercial training.

## Non-goals

- literal language translation;
- veterinary diagnosis or pain inference;
- claiming universal semantics from a small convenience sample;
- inducing avoidable distress to manufacture labelled vocalisations;
- reporting random clip-split performance as evidence of cross-cat generalisation.

## Immediate deliverables

- [ ] licence note for CatMeows and any other external audio source;
- [ ] dataset manifest builder that records cat/session/context grouping keys;
- [ ] frozen grouped split protocol;
- [ ] interpretable feature extractor;
- [ ] majority/metadata/duration+F0/MFCC baselines;
- [ ] calibration + grouped evaluation report;
- [ ] A1.1 unsupervised clustering notebook/script with stability checks;
- [ ] mapping into #18 event records for the future naturalistic pilot.

**Comment by in-c0** [comment]
### Execution status — 2026-08-21

Implemented and merged:
- #24: CatMeows metadata manifest + leakage-safe grouped split tooling
- #25: pYIN/MFCC/interpretable acoustic extraction + frozen cross-cat baseline ladder

In review:
- #27: A1.1 cluster stability + identity/owner/breed/sex confound diagnostics

**Real-data execution is currently blocked only on archive acquisition in this execution environment.** The official Zenodo `dataset.zip` endpoint returned HTTP 429 here. The University of Milan institutional repository independently exposes the same CatMeows full-text archive, but its binary endpoint is also not retrievable from this environment. No unverified mirror has been substituted for a research result.

Canonical dataset facts used by the protocol remain: CatMeows v1.0.2, 440 WAV meows, 21 cats, three supplied emission contexts, Zenodo DOI 10.5281/zenodo.4008297, published archive MD5 `b5fa911bcd6514e39dfef6876f747df4`, scientific-research/non-commercial terms.

Once an archive is locally available, the frozen execution path is:

```bash
python -m audio.catmeows_manifest /path/to/CatMeows --output /tmp/catmeows-manifest.csv
python -m audio.acoustic_features /tmp/catmeows-manifest.csv /path/to/CatMeows --output /tmp/catmeows-features.csv
python -m audio.acoustic_baselines /tmp/catmeows-features.csv --group-key cat_id --output /tmp/a1-baselines.json
python -m audio.cluster_stability /tmp/catmeows-features.csv --feature-set all_acoustic --output /tmp/a1-clusters.json
```

Do not mark A1.0/A1.1 empirical gates complete until those real-data reports exist.

**Comment by in-c0** [comment]
Closed 2026-09-11: the multimodal intent programme was carved out of this repository in #77. This thread's work is preserved unchanged at tag `programme-v0-2026-08-25` and branch `programme/multimodal-intent`. Reopen there if it is ever resumed.
